openlibrary/plugins/openlibrary/js/edit.js

```diff
@@ -458,3 +458,70 @@ export function initEdit() {
         }, 1000);
     }
 }
+
+const isbn_invalid_checksum = 'Invalid Isbn checksum digit';
+const isbn10_wrong_length_or_chars = 'ID must be exactly 10 characters [0-9 or X]. For example 0-19-853453-1 or 0198534531';
+const isbn13_wrong_length_or_chars = 'ID must be exactly 13 characters [0-9]. For example 978-3-16-148410-0 or 9783161484100';
+const invalid_isbn = 'ISBN may be invalid. Click \'Add\' again to submit.'
+
+// a flag to make raiseError perform differently upon subsequent calls
+let addBookWithIsbnErrors = false;
+
+function raiseError(event, errorMessage) {
+    // if first time calling with a given isbn
+    if (!addBookWithIsbnErrors) {
+        addBookWithIsbnErrors = true;
+        const errorDiv = document.getElementById('id-errors');
+        errorDiv.style.display = 'block';
+        errorDiv.innerHTML = errorMessage;
```

Comment:
```suggestion
        errorDiv.classList.remove('hidden');
        errorDiv.textContent = errorMessage;
```
Now that these new divs have the `.hidden` class, we can simply remove the class here instead of modifying the style.
Using `innerHTML` has some [security implications](https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML#security_considerations).  Best to use `textContent` when changing the text of an element.